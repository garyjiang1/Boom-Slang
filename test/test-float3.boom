def testfloat(float a, float b) returns void:
	println(a + b)
	println(a - b)
	println(a * b)
	println(a / b)
	println(a == b)
	println(a == a)
	println(a != b)
	println(a != a)
	println(a > b)
	println(a >= b)
	println(a < b)
	println(a <= b)

float c = 42.0
float d = 3.14159
testfloat(c, d)
testfloat(d, d)
