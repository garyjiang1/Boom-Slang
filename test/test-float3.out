45.1416
38.8584
131.9468
13.3690
false
true
true
false
true
true
false
false
6.2832
0.0000
9.8696
1.0000
true
true
false
false
false
true
false
true
